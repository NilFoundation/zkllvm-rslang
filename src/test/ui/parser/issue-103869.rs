enum VecOrMap{
    vec: Vec<usize>,
    //~^ ERROR expected one of `(`, `,`, `=`, `{`, or `}`, found `:`
    //~| HELP: enum variants can be `Variant`, `Variant = <integer>`, `Variant(Type, ..., TypeN)` or `Variant { fields: Types }`
    //~| ERROR expected item, found `:`
    map: HashMap<String,usize>
}

fn main() {}
