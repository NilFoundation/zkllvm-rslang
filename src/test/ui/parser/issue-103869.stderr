error: expected one of `(`, `,`, `=`, `{`, or `}`, found `:`
  --> $DIR/issue-103869.rs:2:8
   |
LL |     vec: Vec<usize>,
   |        ^ expected one of `(`, `,`, `=`, `{`, or `}`
   |
   = help: enum variants can be `Variant`, `Variant = <integer>`, `Variant(Type, ..., TypeN)` or `Variant { fields: Types }`

error: expected item, found `:`
  --> $DIR/issue-103869.rs:2:8
   |
LL |     vec: Vec<usize>,
   |        ^ expected item

error: aborting due to 2 previous errors

